### ISSUE TITLE
New Source Microsoft Dynamics 365 Business Central

### ISSUE THREAD

**Issue by marcosmarxm** [airbytehq/airbyte#8067]
## Tell us about the problem you're trying to solve
https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/api-reference/v2.0/

## Describe the solution you’d like
A clear and concise description of what you want to see happen, or the change you would like to see

## Describe the alternative you’ve considered or used
A clear and concise description of any alternative solutions or features you've considered or are using today.

## Are you willing to submit a PR?

Remove this with your answer :-)

**Comment by YowanR** [comment]
@btkcodedev Are you still planning to work on this one?

**Comment by YowanR** [comment]
@btkcodedev bump here :)

**Comment by YowanR** [comment]
@btkcodedev bump here :)

**Comment by btkcodedev** [comment]
Yess, For sure via low-code CDK

**Comment by YowanR** [comment]
Assigned to you! Awesome 💪

**Comment by btkcodedev** [comment]
@YowanR Blocker here,
Could you tag someone to help with preparation of oauth2 credentials to pull data,
I've tried it and it requires azure subscription to get the BC API credentials to get working

**Comment by YowanR** [comment]
Hi @btkcodedev ! Thanks for flagging this. @igrankova Are you able to help us unblock?

**Comment by igrankova** [comment]
@btkcodedev, hi!
Can you please give me more details to help you?
What I've already investigated, we don't have a test account for this connector. But we have an Azure account, you can access it using LastPass creds: Microsoft Integration Test Credentials.
Let me know if this may help you or what I can do to unblock you.

**Comment by igrankova** [comment]
@YowanR, we don't have a sandbox account and the OAuth App for Dynamics and it could take time to obtain it. I can create an issue for this and it needs to be prioritized.

**Comment by btkcodedev** [comment]
Blocker Here;
The solution is on hold until 
https://github.com/airbytehq/airbyte/issues/24094 is solved :)

**Comment by YowanR** [comment]
Agreed! Moving this to on-hold. Once it's unblocked, I'll ping you @btkcodedev

**Comment by misteryeo** [comment]
**Issue was linked to Harvestr Discovery:**  [New Source: Microsoft Dynamics Business Central](https://app.harvestr.io/components/0/list/84d_FgfjX)
